gradle-plugin/build.gradle

```diff
@@ -58,7 +56,7 @@ gradlePlugin {
   plugins {
     simplePlugin {
       id = 'org.projectnessie'
-      implementationClass = 'org.projectnessie.quarkus.gradle.QuarkusAppPlugin'
+      implementationClass = 'org.projectnessie.nessierunner.gradle.NessieRunnerPlugin'
       displayName = 'Quarkus App Runner'
       description = 'Start and stop a quarkus app as gradle tasks for integration testing'
```

Comment:
are name and description still accurate?

Reply:
updated